enum.rb

```diff
@@ -0,0 +1,53 @@
+#:nodoc:
+module Enumerable
+  def my_each
+    length.times { |index| yield(self[index]) }
+    self
+  end
+
+  def my_each_with_index
+    length.times { |index| yield(self[index], index) }
+    self
+  end
+
+  def my_select
+    array = []
+    my_each { |index| array << index if yield(index) }
+    array
+  end
+
+  def my_all?
+    new_array = true
+    my_each { |index| break new_array = false unless yield(index) }
+    new_array
+  end
+
+  def my_any?
+    new_array = false
+    my_each { |index| break new_array = true if yield(index) }
+    new_array
+  end
+
+  def my_none?
+    new_array = true
+    my_each { |index| break new_array = false if yield(index) }
+    new_array
+  end
+
+  def my_count(parameter)
+    count = 0
```

Comment:
###  Should have working `my_count`  identical and returns the same thing as ruby's [`count`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-count). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_count.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
```ary = [1, 2, 4, 2]```
```p ary.my_count ``` should return same results as ```p ary.count ```
```p ary.my_count{ |x| x%2==0 }``` should return same results as```p ary.count{ |x| x%2==0 }```